dashboard/src/components/common/Table.tsx

```diff
@@ -1,6 +1,51 @@
-import { Label } from '.';
+import { IconContext } from 'react-icons';
+import {
+  MdAccessTime,
+  MdDataObject,
+  MdDateRange,
+  MdNumbers,
+  MdOutlineToggleOff,
+  MdQuestionMark,
+  MdTextFields,
+} from 'react-icons/md';
 
-export default function Table({ data }: { data: { [key: string]: any }[] }) {
+import { align, HBox, Label } from '.';
+
+function getIconForDataType(dataType: number | undefined) {
+  // https://github.com/brianc/node-pg-types/blob/master/lib/builtins.js#L12-L73
+  switch (dataType) {
+    case 16: // boolean
+      return <MdOutlineToggleOff />;
+    case 20:
+    case 21:
+    case 23:
+    case 1700: // integer
+      return <MdNumbers />;
+    case 25: // text
+    case 1043:
+      return <MdTextFields />;
+    case 3802: // jsonb
+    case 114: // json
+      return <MdDataObject />;
+    case 1082: // date
+    case 1083: // time
+      return <MdDateRange />;
+    case 1114:
+    case 1184: // timestamp
+      return <MdAccessTime />;
+    case 2950: // uuid
+    default:
+      return <MdQuestionMark />;
```

Comment:
Those numbers are coming from here:
 https://github.com/brianc/node-pg-types/blob/8d2200fd9cecf41f40acc1432c01a316178e711f/lib/builtins.js#L2

I think each user defined enum has a specific id and I'm treating everything "else" as unknown variables. I'll check if there's a specific range of values for user defined types.

For UUID case, I didn't find a good icon. So I made it fall back to the default icon.

Reply:
So we can get the user-created type ids with a minor update on [this query](https://dba.stackexchange.com/a/239727)

```sql
SELECT
  pg_type.oid,
  pg_type.typname, 
  pg_enum.enumlabel
FROM
  pg_type 
JOIN
  pg_enum ON pg_enum.enumtypid = pg_type.oid;
```

Probably only need the `pg_type.oid` here and *might* be able to get away with no join like this:

```sql
SELECT pg_enum.enumtypid from pg_enum
```

This could be put into the initialization queries and referenced here with something like:

```suggestion
      return dbEnumIds.includes(dataType) ? <MdSomething /> : <MdQuestionMark />;
```

Reply:
The simpler solution for now may just be to use an appropriate icon for enums and not worry about a mismatch. Do we have a non-enum/non-uuid type in our schemas that would get caught by this `?` path? If not, default to "assume it's an enum" seems reasonable to me?

Reply:
> For UUID case, I didn't find a good icon. So I made it fall back to the default icon.

I think UUID could fall into the `text` category